scripts/build-tools.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 cd tools
-./autogen.sh
-./configure
+
+mkdir build
```

Comment:
Would it be better to name this build_tools ? or build/tools. Where is the "host build" target built ?

Reply:
By 'host' i guess you mean testbench that was added in FW buildsystem, because it was there previously as --enable-library(?) option. It was direct port of that, but I plan to separate testbench/fw builds.

Reply:
Also I can change build dir name to build_tools, however there is nothing else build from this project that's why I named it just build (for fw you have different platfroms/ut/host so it's differnt story there).